.github/workflows/verify.yaml

```diff
@@ -4,7 +4,7 @@ on: [push]
 jobs:
   tests:
     name: Tests
-    runs-on: ubuntu-18.04 # prod = 16 unfortunately
+    runs-on: ubuntu-20.04 # prod = 16 unfortunately
```

Comment:
Did you make this change because the old dependencies are becoming more difficult to use, or just to be on the latest? If prod is 16, it may be wise NOT to upgrade our CI setup to be further from prod.